images/win/scripts/Installers/Install-MysqlCli.ps1

```diff
@@ -11,9 +11,17 @@ $ArgumentList = ("/install", "/quiet", "/norestart")
 Install-Binary -Url $InstallerURI -Name $InstallerName -ArgumentList $ArgumentList
 
 ## Downloading mysql
-$MysqlMajorVersion = (Get-ToolsetContent).mysql.major_version
-$MysqlFullVersion = (Get-ToolsetContent).mysql.full_version
-$MysqlVersionUrl = "https://dev.mysql.com/get/Downloads/MySQL-${MysqlMajorVersion}/mysql-${MysqlFullVersion}-winx64.zip"
+$MysqlVersion = [version](Get-ToolsetContent).mysql.version
+if ($MysqlVersion.Build -lt 0) {
+    $MysqlVersion = New-Object System.Version (
+        $MysqlVersion.Major,
+        $MysqlVersion.Minor,
+        ([version]((Invoke-WebRequest -Uri https://dev.mysql.com/downloads/mysql/$($MysqlVersion.Major).$($MysqlVersion.Minor).html -UseBasicParsing).Content |
```

Comment:
Let's create  a new variable if we use `$($MysqlVersion.Major).$($MysqlVersion.Minor)` multiple times.

`$MysqlVersionMajorMinor = $MysqlVersion.ToString(2)`